### PR TITLE
Create Adam's Poem

### PR DESCRIPTION
**Pull request by Adamcheng88**

### CODE REVIEW

File: Adam's Poem [hunk 1]
```diff
@@ -0,0 +1 @@
+(I didn't actually write one)
```
Comment (davidbtucker) [inline Adam's Poem]: This is almost a good poem. It needs a title.

File: Adam's Poem [hunk 2]
```diff
@@ -0,0 +1,5 @@
+Here's a haiku:
+
+Now we are learning
+How to use Git and GitHub
+It is very fun
```
Comment (davidbtucker) [inline Adam's Poem:5]: Can you add your name at the bottom?

**Review by davidbtucker — CHANGES_REQUESTED** [review]

File: Adam's Poem (hunk 2 above)
Comment (Adamcheng88) [reply Adam's Poem:5]: Done!

**Review by davidbtucker — APPROVED** [review]
Great!

**Pull request merged**